spec/index.html

```diff
@@ -276,7 +276,12 @@ <h3>RDF Blank Nodes</h3>
     <ul>
       <li>The characters <code>_</code> and <code>[0-9]</code> may appear anywhere in a blank node label.</li>
       <li>The character <code>.</code> may appear anywhere except the first or last character.</li>
-      <li>The characters <code>-</code>, <code>U+00B7</code>, <code>U+0300</code> to <code>U+036F</code> and <code>U+203F</code> to <code>U+2040</code> are permitted anywhere except the first character.</li>
+      <li>The characters
+        <code>-</code>,
+        <code class="codepoint">U+00B7</code>,
+        <code class="codepoint">U+0300</code> to <code class="codepoint">U+036F</code>, and
+        <code class="codepoint">U+203F</code> to <code class="codepoint">U+2040</code>
```

Comment:
They don't play like the other characters, so maybe we don't need to display the 96 combining diacritical marks, and can treat them as below?
```suggestion
        the combining diacritical marks (<code class="codepoint">U+0300</code>
                                      to <code class="codepoint">U+036F</code>),
        &#x203F;&nbsp; (undertie, <code class="codepoint">U+203F</code>), and
        &#x2040;&nbsp; (character tie, <code class="codepoint">U+2040</code>)
```